web/packages/teleport/src/Assist/Chat/Avatar.ts

```diff
@@ -0,0 +1,46 @@
+/*
```

Comment:
is this avatar distinct to chat only or can it be moved to its own component in `design`

Reply:
It's distinct to chat